src/entry-point-static/build.tsx

```diff
@@ -1,5 +1,5 @@
-import React from "react";
```

Comment:
In this case, why did we delete this line as well, instead of just the global call directly below?

Reply:
Oh that's just because nothing in this file requires this import anymore.